testing/content/RepresentationShape.ttl

```diff
@@ -49,7 +49,26 @@ shapes:RepresentationShape
 		sh:class ids:Standard ;
 		sh:severity sh:Violation ;
 		sh:message "<https://raw.githubusercontent.com/IndustrialDataSpace/InformationModel/master/testing/content/RepresentationShape.ttl> (RepresentationShape): An ids:representationStandard property must point from an ids:Representation to an ids:Standard."@en ; 
-	] .
+	] ;
+    
+    sh:property [
+		a sh:PropertyShape ;
+		sh:path ids:created ;
+		sh:datatype xsd:dateTimeStamp ;
+        sh:maxCount 1 ;
+		sh:severity sh:Violation ;
+		sh:message "<https://raw.githubusercontent.com/IndustrialDataSpace/InformationModel/master/testing/content/RepresentationShape.ttl> (RepresentationShape): An ids:DigitalContent must have at most one ids:created attribute pointing to a valid xsd:dateTimeStamp."@en ; 
+	] ;
+
+	sh:property [
+		a sh:PropertyShape ;
+		sh:path ids:modified ;
+        sh:maxCount 1 ;
+		sh:datatype xsd:dateTimeStamp ;
+		sh:severity sh:Violation ;
+		sh:message "<https://raw.githubusercontent.com/IndustrialDataSpace/InformationModel/master/testing/content/RepresentationShape.ttl> (RepresentationShape): An ids:modified property of an ids:Representation must have valid xsd:dateTimeStamp values as its object."@en ; 
```

Comment:
Not 100% happy with the error message. One similar to the one above might be better

Reply:
Resolved in dbed026